client/my-sites/plans/current-plan/my-plan-card/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import PlanIcon from 'components/plans/plan-icon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const MyPlanCard = ( { action, isError, isPlaceholder, details, plan, tagLine, title } ) => {
+	const cardClassNames = classNames( 'my-plan-card', {
+		'has-action-only': action && ! details && ! isPlaceholder,
+	} );
+	const detailsClassNames = classNames( 'my-plan-card__details', {
+		'is-error': isError,
+		'is-placeholder': isPlaceholder && ! details,
+	} );
+	const actionClassNames = classNames( 'my-plan-card__action', {
+		'is-placeholder': isPlaceholder && ! action,
+	} );
+	const hasSecondaryArea = details || action || isPlaceholder;
+
+	return (
+		<Card className={ cardClassNames } compact>
+			<div className="my-plan-card__primary">
+				{ plan && (
+					<div className="my-plan-card__icon">
```

Comment:
I wonder if we really need this `div` - could we style the `PlanIcon` inside a `my-plan-card__primary` instead?

Reply:
Definitely 👍